webapp/app.py

```diff
@@ -25,6 +26,17 @@ def index():
     return render_template("index.html")
 
 
+@app.route("/download/<osname>")
+def osredirect(osname):
+    if osname not in ("windows", "macos"):
+        return redirect("/#install", code=301)
```

Comment:
Do we actively want to support `/download/turkey`, which is a meaningless URL? It seems like it would be better to explicitly listen for only `/download/macos` and `/download/windows`. I think it's much neater to always show people 404s unless we actually think we understand what they were looking for.

This could be done either like this:

``` python3
@app.route("/download/regex('windows|macos'):osname>")
def osredirect(osname):
```

Or something like this:

``` python3
@app.route("/download/windows")
@app.route("/download/macos")
def osredirect():
    osname = request.path.split('/')[-1]
```

Reply:
I like the first option.